frontend/src/app/coworking/coworking-home/coworking-home.component.html

```diff
@@ -51,12 +51,12 @@ <h3 matListItemMeta>&rsaquo;</h3>
     <div class="upcoming-card-container">
       <!-- Show upcoming room reservations -->
       <ng-container
-        *ngIf="((upcomingRoomReservation$ | async) ?? []).length > 0">
+        *ngIf="((upcomingReservations$ | async) ?? []).length > 0">
         <div class="reservations-header">
           <h2 class="heading">Upcoming Reservations</h2>
         </div>
         <div class="upcoming-reservations-container">
-          <ng-container *ngFor="let r of upcomingRoomReservation$ | async">
+          <ng-container *ngFor="let r of initUpcomingReservations() | async">
```

Comment:
We should see if we can drop this method call, as well. This is looking really close!